flink-runtime/src/test/java/org/apache/flink/runtime/leaderelection/LeaderChangeClusterComponentsTest.java

```diff
@@ -142,7 +142,7 @@ public void testReelectionOfJobMaster() throws Exception {
 
 		highAvailabilityServices.revokeJobMasterLeadership(jobId).get();
 
-		assertThat(jobResultFuture.isDone(), is(false));
```

Comment:
Is there any case that failed on this check?
Looks to me the issue of FLINK-11524 is happening in a line below `JobResultUtils.assertSuccess(jobResult);` (it used to be `assertThat(jobResult.isSuccess(), is(true));` when the issue was reported)

Reply:
I actually don't remember; maybe I saw it fail in _some_ travis run.